CHANGES/README.rst

```diff
@@ -1,15 +1,23 @@
-.. _Adding change notes with your PRs:
+.. _Making a pull request:
+
+Making a pull request
```

Comment:
@Dreamsorcerer this title was a mistake:
1) Since it's rendered on GitHub in a folder with change notes, on any branch, it shouldn't really mention the PR process that much — just the process for having the changelog.
2) The Sphinx site now has 2 subsequent titles with the same name but different capitalization — https://docs.aiohttp.org/en/stable/contributing.html#making-a-pull-request and https://docs.aiohttp.org/en/stable/contributing.html#id2.